extensions/amp-story/0.1/amp-story.js

```diff
@@ -372,8 +372,13 @@ export class AmpStory extends AMP.BaseElement {
     const gestures = Gestures.get(this.element,
         /* shouldNotPreventDefault */ true);
 
-    gestures.onGesture(SwipeXYRecognizer, () => {
-      if (this.bookend_.isActive()) {
+    gestures.onGesture(SwipeXYRecognizer, e => {
+      const xSwipe = Math.abs(e.data.deltaX);
+      const ySwipe = Math.abs(e.data.deltaY);
+      const minSwipeDelta = 50;
+      const canShowEducationOverlay =
```

Comment:
Avoid the word `education`  as the current terminology is `hint`.

Reply:
this is remove